documents/service.go

```diff
@@ -405,10 +405,26 @@ func (s service) Derive(ctx context.Context, payload UpdatePayload) (Model, erro
 		}
 
 		payload.Collaborators.ReadWriteCollaborators = append(payload.Collaborators.ReadWriteCollaborators, did)
-		if err := doc.(Deriver).DeriveFromCreatePayload(ctx, payload.CreatePayload); err != nil {
-			return nil, errors.NewTypedError(ErrDocumentInvalid, err)
+		for _, a := range payload.CreatePayload.Attributes {
```

Comment:
not sure I understand why the attribute has `template` that holds the template ID